proposals/0000-overloaded-record-fields.rst

```diff
@@ -0,0 +1,359 @@
+.. proposal-number:: Leave blank. This will be filled in when the proposal is
+                     accepted.
+
+.. trac-ticket:: Leave blank. This will eventually be filled with the Trac
+                 ticket number which will track the progress of the
+                 implementation of the feature.
+
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+
+Overloaded Record Fields
+========================
+
+This is a proposal to introduce a new extension,
+``OverloadedRecordFields``, to make it easier to work with record
```

Comment:
See [criticism of the extension name](https://www.reddit.com/r/haskell/comments/518d8q/ghc_proposal_for_overloadedrecordfields_plus_some/d7aca7s) from @Hrothen. I'm rather attached to the current colour of the bikeshed, but perhaps there is an argument to change it.